applications/logging-operator/6.0.3/defaults/cm.yaml

```diff
@@ -152,20 +154,19 @@ data:
         tls.verify: "On"
         K8S-Logging.Parser: "Off"
         K8S-Logging.Exclude: "Off"
-      forwardOptions:
```

Comment:
Can you please explain these changes and provide config references to logging operator documentation?